src/cfn.py

```diff
@@ -82,6 +83,7 @@ def update_template(stackname):
     LOG.info("Create: %s", pformat(delta.plus))
     LOG.info("Update: %s", pformat(delta.edit))
     LOG.info("Delete: %s", pformat(delta.minus))
+    LOG.info("New Terraform generated file: %s", pformat(json.loads(delta.terraform)))
```

Comment:
`update_template` is not the right name anymore as the process is integrating Terraform with its own resource files, not just CloudFormation templates. It was already expanded to also run Salt, and other AWS API calls for S3 and SQS. 

Possible names:

- `reprovision`
- `apply` (of Terraform fame)
- `re_source`
- `update_...` (update what?)
- `source_again` (crappy)

It's important to render this is a human operation, not to be performed at every deploy.

Reply:
`update_infrastructure`

it sounds necessarily big and scary